### PR TITLE
MAINT: update numpy dependency; add Python 3.6

### PR DESCRIPTION
**Pull request by matthew-brett**
This will also trigger tests against the latest wheels.

### CODE REVIEW

**Comment by matthew-brett** [comment]
I think this one is a fairly straightforward win - any takers for a quick review?

File: .travis.yml [hunk 1]
```diff
@@ -23,10 +23,12 @@ env:
         - DEPENDS="numpy scipy matplotlib h5py pillow"
         - PYDICOM=1
         - INSTALL_TYPE="setup"
+        - EXTRA_WHEELS="https://5cf40426d9f06eb7461d-6fe47d9331aba7cd62fc36c7196769e4.ssl.cf2.rackcdn.com"
+        - PIP_FLAGS="--find-links=$EXTRA_WHEELS"
```
Comment (grlee77) [inline .travis.yml]: Is this PIP_FLAGS always included by pip in all cases then?  I was unfamiliar with that environment variable and don't see it mentioned in the manual?  I would have guessed it wanted PIP_FIND_LINKS.

**Comment by grlee77** [comment]
LGTM.  the comment above is mainly a question for my own education

File: .travis.yml (hunk 1 above)
Comment (matthew-brett) [inline .travis.yml]: Oops - sorry - that should be EXTRA_PIP_FLAGS, used explicitly below.  I've changed it...

**Comment by matthew-brett** [comment]
Thanks for the review - merging as I think this stuff is uncontroversial.

**Pull request merged**